static/js/map.js

```diff
@@ -876,8 +876,8 @@ function gymLabel(item) {
 
     var park = ''
     if ((item['park'] !== '0' && item['park'] !== 'None' && item['park'] !== undefined && item['park']) && (noParkInfo === false)) {
-        if (item['park'] === 1) {
-            // RM only stores boolean, so just call it "Park Gym"
+        if (item['park'] === 1 || item['park'] === '1') {
```

Comment:
if item['park'] comes as string it should be converted to intval in the lib file so it becomes a number in js